src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -48,21 +46,21 @@ public CommandBase spinDown() {
 
     /**
      * Finds the needed velocity to hit target (inches) (x,y) with a given launch
-     * angle (theta).
+     * angle (THETA).
      */
 
-    public static double calculateVelocity(double x) {
+    public static double calculateVelocity(final double distanceToTarget) {
         /**
          * xtanθ has to be greater than y- if it isn't then it is impossible to reach
          * that point with given launch angle. Eg. trying to hit a target at (1, 100)
          * with a launch angle of 1 degree.
          */
-        if (x * Math.tan(Math.toRadians(theta)) >= y) {
-            double gravitySide = gravity * x * x;
-            double tanSide = x * Math.tan(Math.toRadians(theta)) - y;
-            double cosSide = Math.cos(Math.toRadians(theta)) * Math.cos(Math.toRadians(theta));
+        if (distanceToTarget * Math.tan(Math.toRadians(THETA)) >= verticalDistance) {
+            final double GRAVITY_SIDE = GRAVITY * distanceToTarget * distanceToTarget;
```

Comment:
Sorry I led you astray here. These don't need to be in all caps. Just the `final static` ones above.